Documentation/running-lkvm-stage1.md

```diff
@@ -136,7 +136,7 @@ bind: No such file or directory
   Warning: Failed init: kvm_ipc__init
 ```
 
-The LKVM stage1 currently requires $HOME to be defined
+The LKVM stage1 currently requires $HOME to be defined and valid
 (see [issue #1393](https://github.com/coreos/rkt/issues/1393)).
 When started from a systemd unit file, use `User=root`.
```

Comment:
I am undecided whether to keep that paragraph. I decided to keep it just in case the user mistakenly starts rkt with `$HOME=/nonexistent-dir` or something... but that's not very plausible so I'm ok to remove these lines if you prefer.